src/App.js

```diff
@@ -1,24 +1,56 @@
 import './App.css';
```

Comment:
General suggestions:
- File structure: create a folder called components to organize all your components there (separate them from other files like firebase.js)
- Favicon: add a custom favicon that reflects what your site does! (upload inside the public folder and replace the react favicon.ico)
- Site title: the title that shows up on the tab can be changed from 'React App' by changing the index.html in public folder (<title> tag)
- README: can add a nice summary of your app to replace the react boilerplate readme text